docs/FAQ.md

```diff
@@ -0,0 +1,76 @@
+# Frequently Asked Questions
+
+
+## Team Questions
+
+### Can undergraduate students not in our research lab be part of a team?
+
+Yes.
+
+### Can recent graduates qualify to be members of a team?
+
+Yes, being a student is not a requirement.
+
+### Is there a constraint on the number of Team members?
+
+Currently, the team limit is 6 members (not counting the advisor(s)).
+
+### Can the team have two advisors?
+
+Yes.
+
+### Can we have more than one team?
+
+Yes, as long as team members belong exclusively to a single team.  However, advisors can advise more than one team.
+
+### Can we collaborate with another University as a single team for the contest?
+
+Yes.
+
+## Contest Objective Questions
+
+### The main objective of the contest appears to optimizing routing speed. However, should the router be timing-driven or wire-length driven? More specifically, should the criticality of nets be computed within the router to influence routing order?
+
+A timing-driven router will not be required, the competition is mostly focused on wirelength driven approaches.  There will be a scoring tool provided that will calculate the critical path via wirelength, however, as noted in the scoring criteria, this critical path length will be a smaller portion of the overall score.  As you have noted, it is primarily focused runtime of generating a legal routing solution.
```

Comment:
```suggestion
Indeed, the biggest focus of this contest is on producing a legal solution within the shortest possible wall-clock time. A secondary, smaller, component of the score is to optimize the critical-path (not the total) wirelength. This metric is similar to the critical-path delay that a timing-driven router would be expected to optimize for, but instead of considering the path with the maximum total logic and net delay from a timing startpoint to a timing endpoint (including through combinatorial elements such as LUTs) only the path with the maximum total net wirelength is considered. The intention is for critical-path wirelength to be much easier to compute, and documentation as well as reference code to do so will be published soon.
```